fieldservice_skill/views/hr_skill.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <menuitem
+        id="menu_hr_skill"
+        name="Skills"
+        action="hr_skills.hr_skill_view_form"
```

Comment:
@brian10048 Can you please improve the action name or create new action of skills and bind it here?